modules/ddl-and-loading/pages/creating-a-loading-job.adoc

```diff
@@ -763,57 +763,74 @@ The timestamp parameter should be in one of the following formats:
+A reducer function aggregates multiple values of a non-ID attribute into one attribute value of a single vertex or edge.
+Reducer functions are computed incrementally; that is, each time a new input token is applied, a new resulting value is computed.
 
-To reduce and load aggregate data to an attribute, the attribute expression has the form
+To reduce and load aggregate data to an attribute, the attribute expression has the following form:
 
 [source,gsql]
 ----
 REDUCE(reducer_function (input_expr))
 ----
 
-where _reducer_function_ is one of the functions in the table below. _input_expr_ can include non-reducer functions, but reducer functions cannot be nested.
+`reducer_function` is one of the functions in the following table.
+`input_expr` can include non-reducer functions, but reducer functions cannot be nested.
 
-Each reducer function is overloaded so that one function can be used for several different data types.For primitive data types, the output type is the same as the _input_expr_ type.For LIST, SET, and MAP containers, the _input_expr_ type is one of the allowed element types for these containers (see "Complex Types" in the Attribute Data Types section).The output is the entire container.
+Each reducer function is overloaded so that one function can be used for several data types:
+
+* For primitive data types, the output type is the same as the type of the input expression `input_expr`.
+* For `LIST`, `SET`, and `MAP` containers, the _input_expr_ type is one of the xref:system-and-language-basics.adoc#_collection_types[allowed element types] for these containers.
+The output is the entire container.
 
 |===
-| Function name | Data type of _arg_: Description of function's return value
+| Function name | Return value
 
-| max(_arg_)
-| INT, UINT, FLOAT, DOUBLE: maximum of all _arg_ values cumulatively received
+| `max(arg)`
+| `INT`, `UINT`, `FLOAT`, `DOUBLE`: maximum of all `arg` values cumulatively received
 
-| min(_arg_)
-| INT, UINT, FLOAT, DOUBLE: minimum of all _arg_ values cumulatively received
+| `min(arg)`
+| `INT`, `UINT`, `FLOAT`, `DOUBLE`: minimum of all `arg` values cumulatively received
 
-| add(_arg_)
-| INT, UINT, FLOAT, DOUBLE: sum of all _arg_ values cumulatively received  STRING: concatenation of all arg values cumulatively received  LIST, SET element: list/set of all _arg_ values cumulatively received  MAP (key -> value) pair: key-value dictionary of all key-value pair _arg_ values cumulatively received
+| `add(arg)`
+a| * `INT`, `UINT`, `FLOAT`, `DOUBLE`: sum of all `arg` values cumulatively received
+* `STRING`: concatenation of all arg values cumulatively received
+* `LIST`, `SET` element: list/set of all `arg` values cumulatively received
+* `MAP` (key -> value) pair: key-value dictionary of all key-value pair `arg` values cumulatively received.
```

Comment:
Should we mention that merging also includes key-value pairs that already exist in TigerGraph?